testcase/discover.go

```diff
@@ -33,10 +33,12 @@ func discoverTestDirectory(path string) ([]TestCaseSet, error) {
 	}
 	var result []TestCaseSet
 	for _, f := range files {
-		if f.IsDir() || !strings.HasSuffix(f.Name(), ".json") {
+		log.Debugf("Read file: %s", f.Name())
+		if f.IsDir() || (!strings.HasSuffix(f.Name(), ".json") && !strings.HasSuffix(f.Name(), ".yaml")) {
 			continue
 		}
 		fullpath := filepath.Join(path, f.Name())
+		log.Debugf("Read file fullpath: %s", fullpath)
```

Comment:
Leftover debug prints (lines 36 and 41)? The path to the file being read is also logged in NewFromFile (testcase.go). It should be enough to log the filename in a single place, but I don't have a particular opinion of where.

Reply:
You are right, but because is not on the same function, we can keep the two.